tests/spread/general/run-user/rockcraft.yaml

```
@@ -0,0 +1,23 @@
+name: run-user-test
+version: latest
+summary: A ROCK with a nonroot user
+description: A ROCK that has a nonroot default OCI user
+license: Apache-2.0
+base: ubuntu:22.04
+run-user: _daemon_
+services:
+  test:
+    override: replace
+    command: /usr/bin/sleep 10
+    startup: enabled
+    on-success: shutdown
+    on-failure: shutdown
+platforms:
+  amd64:
+
+parts:
+  my-part:
+    plugin: nil
+    overlay-script: |
+      set -x
+      useradd -R $CRAFT_OVERLAY -M -U -r test-user
```

```suggestion
      useradd -R $CRAFT_OVERLAY -M -U -r test-user
      
```
(adds newline at the end of the file :slightly_smiling_face: )

---

;)